### DISCUSSION TITLE
Best practices for 3rd party HTTP calls

### DISCUSSION THREAD

**Discussion by MCKRUZ** [jasontaylordev/CleanArchitecture#1084] [Q&A]
I'm looking to create a call out to a Databricks instance as part of a project using the clean architecture template.  I was wondering what everyones thoughts were on:

1.) Where this should go?
2.) Any thoughts on best practices?

**Comment by DennisJansenDev** [comment]
The implementation of the databricks connection should be put in your infrastructure layer. And you define an interface for it in your application layer, if needed.
